src/analysis/exploration/findBlockedReaction.m

```diff
@@ -33,7 +33,7 @@
     end
 else
     solution = solveCobraLPCPLEX(model, 0, 0, 0, [], 1e-6);
-    blockedReactions = model.rxns(find(abs(solution.full) < 1e-11))';
+    blockedReactions = model.rxns(abs(solution.full) < 1e-11)';

```

Comment:
I would still argue, that your "null criterion" should depend on the requested feasibility Tolerance used. 
if you have a feasibility tolerance of only 1e-4 than a reaction carrying a flux of 1e-5 can still be blocked. 
Or, since, it is also a fixed number above, at least the same tolerance (1e-10) should be used.